packages/autorest.python/autorest/black/__init__.py

```diff
@@ -8,13 +8,15 @@
 import os
 from typing import Any, Dict
 import black
+from black.mode import Mode as BlackMode  # pylint: disable=no-name-in-module
```

Comment:
The pyright reported error like below for the original code:
![image](https://user-images.githubusercontent.com/59815250/200242376-a5722b2b-96e1-46ed-95e9-702b1a868f07.png)

I updated to use `# pyright: ignore [reportPrivateImportUsage]` on this line instead of introducing a pylint-disable.